db/scripts/sanitise.sql

```diff
@@ -0,0 +1,33 @@
+DELETE FROM "sessions";
+DELETE FROM "audits";
+DELETE FROM "blazer_audits";
+DELETE FROM "validation_errors";
+DELETE FROM "apply_application_sync_requests";
+
+-- Users
+UPDATE "users"
+SET
+  dfe_sign_in_uid = null,
+  dttp_id = null,
```

Comment:
Can we retain the original `ids` these as it is not really needed to be sanitised.

Reply:
+1